keel-ec2-api/src/main/kotlin/com/netflix/spinnaker/keel/api/ec2/RedirectConfig.kt

```diff
@@ -0,0 +1,10 @@
+package com.netflix.spinnaker.keel.api.ec2
+
+data class RedirectConfig(
```

Comment:
`RoutingConfig`? Technically, there's no redirect involved in the load balancer routing traffic.